tests/config.mk

```diff
@@ -0,0 +1,14 @@
+SHELL := /bin/bash
+
+TFVARS_PATH := terraform/modules/config/local.tfvars
+
+ifeq ($(wildcard ${TFVARS_PATH}),)
+  RESOURCE_NAME :=
+  ACCESS_KEY_LOCAL :=
+else
+  export RESOURCE_NAME := $(shell sed -n 's/resource_name *= *"\([^"]*\)"/\1/p' ${TFVARS_PATH})
+  export ACCESS_KEY_LOCAL := $(shell sed -n 's/access_key_local *= *"\([^"]*\)"/\1/p' ${TFVARS_PATH})
```

Comment:
Is this `access_key_local` var in the tfvars file used by terraform at all? If not, we should probably get the local access key a different way so that we don't have to add it to the terraform config vars.

Reply:
@rancher-max  makes total sense, i forgot this one there.